file-formats/siad/type/zif_aff_siad_v1.intf.abap

```diff
@@ -0,0 +1,43 @@
+INTERFACE zif_aff_siad_v1
+  PUBLIC.
+
+  TYPES:
+
+    "! <p class="shorttext">Assignment</p>
+    "! Assignment
+    BEGIN OF ty_assignment,
+
+      "! <p class="shorttext">Business Role Template ID</p>
+      "! Business role template ID
+      "! $required
+      business_role_template_id   TYPE c LENGTH 30,
```

Comment:
Is fine for me. Just two minor remarks: 

1. In most cases, we call it object **names** (instead of their ID). Maybe, we can even skip it at all (I guess this would be my preferred solution) However, I would take this to the UX review what they think.
2. As mentioned before, for object names with the length of 30 we already provide a generic type which you can reuse: `zif_aff_types_v1=>ty_object_name_30` (or `if_aff_types_v1=>ty_object_name_30` within SAP systems)